src/styles/SearchResultsStyle.css

```diff
@@ -0,0 +1,19 @@
+.card-image{
```

Comment:
If you have common properties multiple classes share its a good idea to group them together to save repetition, eg:
```
.card-image,
.paragraph,
.arranged-images {
  display: flex;
}
```